frontend/src/pages/home/components/Slogan/Slogan.tsx

```diff
@@ -1,30 +1,60 @@
+import { useEffect, useState } from 'react';
+
 import styled from '@emotion/styled';
 import { useNavigate } from 'react-router-dom';
 
+import { getMineMentoring } from '../../../../common/apis/getMineMentoring';
 import { useAuth } from '../../../../common/components/AuthProvider/AuthProvider';
 import Button from '../../../../common/components/Button/Button';
 import { PAGE_URL } from '../../../../common/constants/url';
 
 function Slogan() {
   const { authenticated } = useAuth();
   const navigate = useNavigate();
+  const [myMentoringId, setMyMentoringId] = useState<null | number>(null);
 
   const handleMentoringCreation = () => {
     if (authenticated) {
+      if (myMentoringId !== null) {
+        navigate(PAGE_URL.CREATED_MENTORING);
+        return;
+      }
       navigate(PAGE_URL.MENTORING_CREATE);
     } else {
       navigate(PAGE_URL.LOGIN);
     }
   };
```

Comment:
p4: 컨벤션에 맞게 얼리 리턴 사용하는건 어때??
```suggestion
 const handleMentoringCreation = () => {
  if (!authenticated) return navigate(PAGE_URL.LOGIN);
  if (myMentoringId) return navigate(PAGE_URL.CREATED_MENTORING);
  navigate(PAGE_URL.MENTORING_CREATE);
};

```

Reply:
좋아👍👍 반영할게!